libs/cua-driver-rs/scripts/install.ps1

```
@@ -0,0 +1,595 @@
+        $v = $Release -replace '^v', ''
+        Write-Step "using -Release $v"
+        return $v
+    }
+    Write-Step "resolving latest $TagPrefix* release via GitHub API"
+    # Pull the first 40 releases so we still find the latest one even when
+    # several unrelated tag prefixes have shipped recently. The cua-driver-rs
+    # tag prefix is distinct from the Swift cua-driver tag prefix (one extra
+    # "-rs-"), so the simple StartsWith filter is unambiguous.
+    $releases = Invoke-RestMethod -Uri "https://api.github.com/repos/$Repo/releases?per_page=40" `
+                                  -UseBasicParsing
+    $matches = $releases | Where-Object { $_.tag_name -like "$TagPrefix*" }
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)